src/components/PurchaseOrder/PO.js

```diff
@@ -137,8 +153,16 @@ class PO extends Component {
   }
 
   onAddPOLine = (e) => {
+    const { resources } = this.props;
+    const linesLimit = Number(get(resources, ['linesLimit', 'records', '0', 'value'], '1'));
+    const poLines = get(resources, ['order', 'records', '0', 'po_lines'], []);
+
     if (e) e.preventDefault();
```

Comment:
I'm not sure if you need to pass `e` in this function and run this line. The question is - what is trying to be prevented here?